fdmi_app_hello

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python3
```

Comment:
This is changed or copied from the original "fdmi_app", right?
What is changed from the original version?

We should avoid duplicating code in this way.